packages/excalidraw/element/binding.ts

```diff
@@ -828,6 +828,9 @@ const bindingBorderTest = (
   { x, y }: { x: number; y: number },
   app: AppClassProperties,
 ): boolean => {
+  if (!element || !element.width || !element.height) {
```

Comment:
why check for width/height?

Reply:
probably not necessary. I just checked for all the attributes used below.